### PR TITLE
Remove dead code: legacy read/write ops

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by bartlomieju** [comment]
@ry in `//js/files.ts` - `readSync` and `writeSync` still use flatbuffers

**Comment by ry** [comment]
@bartlomieju thanks - fixed.

File: cli/ops/dispatch_minimal.rs [hunk 1]
```diff
@@ -72,21 +69,23 @@ fn test_parse_min_record() {
   assert_eq!(parse_min_record(&buf), None);
 }
 
-pub fn dispatch_minimal(
+pub type MinimalOp = dyn Future<Item = i32, Error = ErrBox> + Send;
+pub type Dispatcher = fn(i32, Option<PinnedBuf>) -> Box<MinimalOp>;
```
Comment (bartlomieju) [inline cli/ops/dispatch_minimal.rs]: I'm not a fan of this name. I always imagined "dispatcher" as a function which maps "op id" to "op handler".

Comment (ry) [reply cli/ops/dispatch_minimal.rs]: that's what this does (modulo the op id part)

Comment (bartlomieju) [reply cli/ops/dispatch_minimal.rs]: Mmmm, I'm not sure. It looks like `Dispatcher` is actually an "op", it returns `MinimalOp` which is the result of that op. I had a feeling that it was supposed to be that `Dispatcher` selects `MinimalOp` which returns some `MinimalOpResult`. But that's nomenclature... just thinking out loud

Comment (ry) [reply cli/ops/dispatch_minimal.rs]: Op = Promise = Future .. there are many ops. An OpDispatcher (yet to be defined but outlined here https://github.com/denoland/deno/issues/2730) is something that returns an Op.

So I think the term Dispatcher is appropriate here. That said, I consider this bit ephemeral as I plan on adding the OpDispatcher trait soon.

**Pull request merged**